src/accidental.ts

```diff
@@ -528,17 +565,17 @@ export class Accidental extends Modifier {
     L('Rendering: ', type, accX, accY);
 
     if (!cautionary) {
-      glyph.render(context, accX, accY);
+      glyph.render(ctx, accX, accY);
     } else {
       // Render the accidental in parentheses.
-      parenRight.render(context, accX, accY);
-      accX -= getGlyphWidth(parenRight);
+      check<Glyph>(parenRight).render(ctx, accX, accY);
+      accX -= check<Glyph>(parenRight).getMetrics().width ?? 0;
```

Comment:
the `?? 0` is a silent failure again -- the previous code expects it to be defined, no?

Can we remove the `?? 0` here and in the other `getGlyphWidth` replacements?

Reply:
This is no longer required, width is now not optional. Removed.